src/store/bitpay-id/bitpay-id.effects.ts

```diff
@@ -524,6 +560,26 @@ export const startFetchDoshToken = (): Effect => async (dispatch, getState) => {
   }
 };
 
+export const startFetchSupportedCurrencies =
+  (token: string): Effect =>
+  async (dispatch, getState) => {
+    try {
+      const {APP} = getState();
+      const {data} = await UserApi.fetchSupportedCurrencies(token);
+
+      dispatch(
+        BitPayIdActions.successFetchSupportedCurrencies(
+          APP.network,
+          data.user.currencies,
+        ),
+      );
+    } catch (err) {
+      dispatch(LogActions.error('Failed to fetch supported currencies'));
+      dispatch(LogActions.error(JSON.stringify(err)));
```

Comment:
We should handle this kind of errors as `Error` type instead to assume it's a string. 

```
if (err instanceof Error) {
         errMsg = `${err.name}: ${err.message}`;
       } else {
         errMsg = JSON.stringify(err);
       }
```